advocacy_docs/dev-guides/deploy/windows.mdx

```diff
@@ -0,0 +1,149 @@
+
+### Connect with psql
+
+Open a new command prompt, and run
+
+```
+psql -U postgres
+___OUTPUT__
+Password for user postgres:
+
+psql (16.4)
+WARNING: Console code page (437) differs from Windows code page (1252)
+         8-bit characters might not work correctly. See psql reference
+         page "Notes for Windows users" for details.
+Type "help" for help.
+
+postgres=#
+```
+
+When prompted, enter the default password (`postgres`). 
+
+!!! Warning
+We're setting up an environment for local development, and have no intention of enabling remote connections or working with sensitive data - so leaving the default password is fine. On a production or multi-user system, or when working with sensitive data, you should *at minimum* set a strong password for the superuser - and realistically, should probably disable password authentication for it entirely.
+!!!
+
+It's a good practice to develop with a user and database other than the default - this helps avoid issues later on when you go to deploy and have no idea what permissions your app needs! So let's start off right by creating an app user and giving it its own database:
+
+```
+create user myapp with password 'app-password';
+create database appdb with owner myapp;
+__OUTPUT__
+CREATE ROLE
+CREATE DATABASE
+```
+
+Now let's quit psql and connect with pgAdmin as our new user. Run,
+
+```
+\q
+```
+
+### Connect with pgAdmin
+
+Now launch pgAdmin via the Start menu.
+
+![pgAdmin 4 default view - server groups on left, welcome message on right](images/pgadmin-default.png)
+
+If you poke around a bit, you'll see that pgAdmin comes with a default connection configured for the `postgres` user to the `postgres` database in our local PostgreSQL server.
+
+Let's add a new connection for our app user to our app database.
+
+1. Right-click the Servers entry on the left, select Register and click Server:
+
+  ![pgadmin Servers group menu with register item and server submenu item selected](images/pgadmin-register-server.png)
+
+2. On the General tab, give it a descriptive name like "myapp db"
+
+3. Switch to the Connection tab:
+
+   enter connection information, using the new role and database we created in psql above:
+   
+   - `localhost` for Host name/address
+   - `appdb` for Maintenance database
+   - `myapp` for Username
+   - `app-password` for Password
+  
+  ...and check the *Save password?* option and click the Save button.
+
+    ![pgadmin screen showing two server connections configured under Servers group on left, with PostgreSQL 16 disconnected and myapp db connected and selected, with activity charts for myapp db shown on right](images/pgadmin-connected.png)
+
+You'll note that some actions that require system-level permissions (e.g., logs) are unavailable, as you're not connected as a superuser. For these, you can use the default superuser connection.
```

Comment:
Suggest just "Note that some actions that require..." instead of "You'll note that some actions that require..."